packages/expo-web-browser/android/src/main/java/expo/modules/webbrowser/error/PackageManagerNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package expo.modules.webbrowser.error;
+
+public class PackageManagerNotFoundException extends Exception {
```

Comment:
As you've already added `expo-errors` try to extend base exception from that module.